src/api/app/controllers/source_controller.rb

```diff
@@ -1560,11 +1560,11 @@ def package_command_linktobranch
 
     # convert link to branch
     rev = ''
-    if not pkg_rev.nil? and not pkg_rev.empty?
+    if !pkg_rev.nil? && !pkg_rev.empty?
```

Comment:
Yes, when `pkg_rev` is `" "`.

@bgeuken  `present?` is equal to `!blank?` but `blank?` is not the same than `empty?` + `nil?` as they behave different for the the string `" "`.

But can we really have `" "` as param? And even if we can, do we care? I mean if everything works for `" "` as well then we can just change it by `present?` as @mdeniz proposed. I don't like the idea of using `strip` to simplify the condition.